composer.json

```diff
@@ -70,13 +70,13 @@
         "paragonie/constant_time_encoding": "^2.2",
         "patchwork/utf8": "^1.2",
         "phpspec/php-diff": "^1.0",
-        "phpunit/php-token-stream": "^1.4 || ^2.0 || ^3.0",
+        "phpunit/php-token-stream": "^1.4 || ^2.0 || ^3.0 || ^4.0",
```

Comment:
Do we really need version 1 and 2 anymore?

Reply:
Yes, bugfix versions should not require higher major dependency versions IMO (unless necessary).
If we don’t need versions 1 and 2 we should remove that in 4.11 I think.